spec/telegram/bot/updates_controller_spec.rb

```diff
@@ -107,7 +108,7 @@ def action(*args)
 
       context 'when action_missing defined' do
         before do
-          controller.class_eval do
+          controller_class.class_eval do
```

Comment:
What is this one for?

Reply:
Sorry, I've started not correct rspec and had error without that change, but now everything is fine